tools/fw-logger/readme.md

```diff
@@ -9,6 +9,7 @@ In order to run this, ensure that your camera is streaming. This can be done usi
 |Flag   |Description   |Default|
 |---|---|---|
 |`-l <xml-path>`|xml file ful path, used to parse the logs||
+|`-s <sleep-duration-in-ms>`|sleep duration between each log message polling| 25 ||
```

Comment:
Please rename it to "polling interval" and adjust description. You can also specify the range for convenience [1-100]

Reply:
done (range is now 25-300ms)